timebinmgr.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TIMEBINMGR_H
+#define TIMEBINMGR_H
+/* This file manages the integer timeline, 
+ * and converts from integers ti to double loga.*/
+
+/*!< The simulated timespan is mapped onto the integer interval [0,TIMEBASE],
```

Comment:
I am a bit confused if the outputs are not uniformly distributed, does it mean the dloga per integer step is not uniform between snapshot files?